ui/src/components/Search.vue

```diff
@@ -9,20 +9,49 @@
     type="text"
     hint=" "
     persistent-hint
+    single-line
+    outlined
+    dense
     @click:append-outer="search"
     @click:clear="clear"
     @keyup.enter="search"
     @focus="isFocused = true"
     @blur="isFocused = false"
   >
-    <template v-slot:message="{ key, message }">
-      <span>{{ message }}</span>
-      <v-btn href="/search-help" target="_blank" color="primary" text x-small>
-        View search syntax
-        <v-icon x-small right>
-          mdi-help-circle-outline
-        </v-icon>
-      </v-btn>
+    <template v-slot:prepend v-if="filterSelector">
```

Comment:
Can we improve the style of the search box?. In my opinion, it's very big if you compare it with other components of the page. In fact its size is the same or similar of the `Indiviuals` and `Organizations` table. Maybe reducing the size and the font might work.

Reply:
I reduced the font and the overall size of the box a bit, let me know if I can do anything else to improve it further.